### ISSUE TITLE
Can this be used with 3rd party keyboard for OSX?

### ISSUE THREAD

**Issue by parnurzeal** [tekezo/Karabiner#797]
Right now, I am trying to use all-in-one Gmyle bluetooth keyboard (coming w/ trackpad) with Macbook Air. But in Karabiner, there is no setting to change a keyboard device. 

Is it fixed to just the laptop keyboard? or there is any hack way to make it be able to set 3rd party keyboard connecting to the laptop?

**Comment by dunkarooftop** [comment]
Yes, it does work with 3rd party keyboard by default. You could also set any mod to only work on certain keyboard too.

https://pqrs.org/osx/karabiner/xml.html.en

Are you having problem?
